specs/EnableInputPassthrough.md

```diff
@@ -0,0 +1,102 @@
+# Background
+WebView2 .NET developers often encounter issues where certain default events are not raised in the WebView2 .NET control. This issue occurs because user inputs are directly delivered to the browser, and the host application does not receive the corresponding messages. This causes some default event handling API of .NET control not to work, such as PreProcessMessage and ProcessCmdKey in WinForms. It also prevents some keys from reaching Key Event realated APIs such as OnKeyDown and ToolStripMenuItem. In order to solve this type of issue, we provide an api which allows user inputs pass through the browser to the host app.
```

Comment:
Please limit line lengths to 80 or 100 characters to make it easier for reviewers reading in GitHub's view.

Reply:
Ok, Modified line length.